### ISSUE TITLE
Allow using existing imagePullSecrets

### ISSUE THREAD

**Issue by sbrunk** [jupyterhub/zero-to-jupyterhub-k8s#1344]
Right now it's possible to specify [credentials for a private docker registry](https://zero-to-jupyterhub.readthedocs.io/en/latest/reference.html?#hub-imagepullsecret) via `hub.imagePullSecret` and `singleuser.imagePullSecret`. This will create a new Secret from the given credentials and use it for pulling images.

In our setup we already have an existing Secret in our cluster we'd like to use as imagePullSecret. I'd be happy to create a PR adding this but I'd like to discuss how this could be configured first.

Here's the current block in `values.yaml` for specifying an pull secret (one for the hub and one for singleuser). 
```yaml
  imagePullSecret:
    enabled: false
    registry:
    username:
    email:
    password:
```

I guess one way would be to add an `existing` field and then set the pull secrets accordingly if `enabled` is false and `existing` is set to some value. What do you think?

**Comment by LaurentGoderre** [comment]
I open #1426 for this because we need this too!

**Comment by minrk** [comment]
closed by #1426
